README.md

```diff
@@ -1,164 +1,76 @@
-# OpenPubkey Reference Implementation
+# OpenPubkey
 
-This repo contains the current reference implementation of OpenPubkey. The reference implementation is a work in progress.
+## Overview
 
-## What is OpenPubkey?
+OpenPubkey allows any OpenID Connect (OIDC)-speaking identity provider to attest to cryptographic public keys alongside identities. This establishes cryptographic signing keys at scale for users and workloads, in a manner that is scalable for workloads and seamless and invisible to users. OpenPubkey does not require any changes to OpenID Connect so it works with today’s OpenID providers.
```

Comment:
Thanks for making this PR. The first few sentence of the readme are the most important as it tells tell someone visiting this repo what OpenPubkey does and why they should care. We should make every word count. I think we should  workshop this a bit.

While they are far from perfect I do prefer the original two sentences as I think they do a better job communicating what OpenPubkey does.

"OpenPubkey adds user or workload generated public keys to OpenID Connect (OIDC) enabling identities to sign messages or artifacts under their OIDC identity. In essence, OpenPubkey is a protocol for getting OpenID Providers (OPs) to bind identities to public keys."

The new description doesn't seem to be as clear. For instance:

> OpenPubkey allows any OpenID Connect (OIDC)-speaking identity provider to attest to cryptographic public keys alongside identities.

* Whose public keys? A user? An OP? 

* Why the word cryptographic here? 

* The OP doesn't know about the public key so it can't attest to the public key.

* It reads me like this is a library for Identity Providers, not parties relying on identity providers.

I'm not sure what the following sentence adds. I would cut it.

> This establishes cryptographic signing keys at scale for users and workloads, in a manner that is scalable for workloads and seamless and invisible to users.

Reply:
Thanks for the collaboration on this Ethan. Updated the "overview" and "how does this work" sections. [Update README.md](https://github.com/openpubkey/openpubkey/pull/83/commits/85ca102ebe604104a3bfdaeeb88dcf6e05c794df)